composer.json

```diff
@@ -102,6 +102,7 @@
         "knplabs/knp-menu-bundle": "^3.2",
         "lcobucci/jwt": "^4.1.5",
         "league/flysystem": "^1.1.4",
+        "league/flysystem-cached-adapter": "^1.1.0",
```

Comment:
This should be also done in framework package

Reply:
ok